robowflex_library/scripts/cob4_test.cpp

```diff
@@ -0,0 +1,105 @@
+/* Author: Juan D. Hernandez */
+
+#include <robowflex_library/log.h>
+#include <robowflex_library/detail/cob4.h>
+#include <robowflex_library/builder.h>
+#include <robowflex_library/geometry.h>
+#include <robowflex_library/io/broadcaster.h>
+#include <robowflex_library/io/visualization.h>
+#include <robowflex_library/planning.h>
+#include <robowflex_library/robot.h>
+#include <robowflex_library/scene.h>
+#include <robowflex_library/trajectory.h>
+#include <robowflex_library/util.h>
+
+using namespace robowflex;
+
+/* \file cob4_visualization.cpp
+ * A simple script that demonstrates how to use RViz with Robowflex with the
+ * COB4 robot. See https://kavrakilab.github.io/robowflex/rviz.html for how to
+ * use RViz visualization. Here, the scene, the pose goal, and motion plan
+ * displayed in RViz.
+ */
+
+static const std::string LEFT_ARM = "arm_left";
+static const std::string RIGHT_ARM = "arm_right";
+
+int main(int argc, char **argv)
+{
+    // Startup ROS
+    ROS ros(argc, argv);
+
+    // Create the Care-O-Bot4 robot.
+    auto cob4 = std::make_shared<Robot>("cob4");
+    cob4->initialize("package://robowflex_resources/cob/robots/cob4-8.urdf.xacro",  // urdf
+                     "package://robowflex_resources/cob/config/cob4-8.srdf",        // srdf
+                     "package://robowflex_resources/cob/config/joint_limits.yaml",  // joint limits
+                     "package://robowflex_resources/cob/config/kinematics.yaml"     // kinematics
+    );
```

Comment:
Why not use the detail implementation?

Reply:
Forgot to do this after creating the detail implementation. Done!